pagopa/io-app
# [IODPAY-196] update new initiative placeholder screen

## Short description
 Updated `InitiativeDetailsScreen.tsx` to follow the new design.


<img width="1091" alt="Screenshot 2023-05-02 at 15 44 12" src="https://user-images.githubusercontent.com/60693085/235684796-5ae98d20-4f3f-43cf-ba86-1acccaadc186.png">


## List of changes proposed in this pull request
- added required SVG to `pictogram` component
- reworked InitiativeDetails to follow design

## How to test
From the wallet, navigate to a not configured initiative's page and make sure the new component renders correctly both in a normal and a small viewport.
#### side note: 
this PR can also be tested with `io-dev-server` 🎉 
just run the app pointing at the dev server and navigate to the wallet screen, you'll find an empty initiative that will display this component.
